eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -175,7 +175,7 @@ function ParseNPMPackage($pkg, $workingDirectory) {
     $releaseNotes = &"${PSScriptRoot}/../Extract-ReleaseNotes.ps1" -ChangeLogLocation $changeLogLoc
   }
 
-  $readmeContentLoc = @(Get-ChildItem -Path $workFolder -Recurse -Include "README.md")[0]
+  $readmeContentLoc = @(Get-ChildItem -Path $workFolder -Recurse -Include "README.md") | Select-Object -Last 1
```

Comment:
I'm not sure that selecting the last one is better or worse the taking the first one. Should we instead probe for where we expect it to be and then use that path? Similar for the ChangeLog.

Reply:
We're recursing, and gci returns the closest to surface _last_ as a result. WE don't probe where we expect it to be for changelog currently, it's actually the same as this one barring the fact that there is a single changelog, but multiple readmes.

![image](https://user-images.githubusercontent.com/45376673/85340384-db928f80-b49a-11ea-9d46-52972f331ce8.png)

Reply:
I was just suggesting we might want to probe in both the README and CHANGELOG case. That seems like it might be a little more predictable then depending on the behavior of gci and allowing additional files in the package cause this to break in strange ways.

Reply:
That's true, this solves 99% of our issues and reduces my manual effort by a huge amount. I'd like to just merge this please.

Reply:
I didn't want to introduce another refactor change to break the world AGAIN this July release. Here is the [issue](https://github.com/Azure/azure-sdk-tools/issues/733). WIll be done by next release. 

Intent is to have the onboarding done this release, I don't want to have to retrod known "ok" parts of the release while I add more features.